common/playbar.js

```diff
@@ -431,23 +431,24 @@ async function saveLaneChanges () {
   await updateSpine(bucket, name, lane.left, lane.right);
 }
 
-function updateSpine (bucket, name, left, right) {
+async function updateSpine (bucket, name, left, right) {
   const input = {
     bucket: bucket,
     name: name,
     left: left,
     right: right
   };
-  lambda.invoke({
+  const lambda = getLambda();
+  await lambda.invoke({
```

Comment:
I see no promise that you are awaiting. Perhaps you meant to [turn the callback into a promise](https://zellwk.com/blog/converting-callbacks-to-promises/)?

Reply:
BTW, if you make `updateSpine` return a promise, then it does not need to itself be `async`. Functions only need to be `async` if they, themselves, use `await`.

Reply:
Removed all that junk.